cmd/juju/bundle/exportbundle.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+package bundle
+
+import (
+	"github.com/juju/cmd"
+	"github.com/juju/gnuflag"
+	"gopkg.in/juju/names.v2"
+
+	"github.com/juju/errors"
+	"github.com/juju/juju/api/bundle"
+	"github.com/juju/juju/apiserver/params"
+	"github.com/juju/juju/cmd/modelcmd"
+	"github.com/juju/juju/cmd/output"
+)
+
+// NewexportbundleCommand returns a fully constructed exportbundle command.
+func NewexportbundleCommand() cmd.Command {
+	return modelcmd.Wrap(&exportBundleCommand{}, modelcmd.WrapSkipModelFlags)
+}
+
+type exportBundleCommand struct {
+	modelcmd.ModelCommandBase
+	out cmd.Output
+	api ExportbundleModelAPI
+}
+
+const exportBundleHelpDoc = `
+Exports the current model configuration into a YAML file.
+
+Examples:
+
+    juju export-bundle
+`
+
+// Info implements Command.
+func (c *exportBundleCommand) Info() *cmd.Info {
+	return &cmd.Info{
+		Name:    "export-bundle",
+		Purpose: "Exports the current model configuration in a YAML file.",
```

Comment:
I'd avoid saying "YAML file" and instead say "charm bundle".

Reply:
done